fuse/data/augmentor/augmentor_toolbox.py

```diff
@@ -452,3 +451,39 @@ def aug_op_batch_mix_up(aug_input: Tuple[Tensor, Tensor], factor: float) -> Tupl
     img = img * (1.0 - factor) + factor * img_mix_up
     labels = labels * (1.0 - factor) + factor * labels_mix_up
     return img, labels
+
+
+def aug_op_random_crop_and_resize(aug_input: Tensor,
+                                  out_size,
```

Comment:
missing type annotation - you have more though - can you please go over the new code and add type annotations?